### PR TITLE
ui: allow user to provide MAC address

### PR DESCRIPTION
**Pull request by masco-enovance**

### CODE REVIEW

File: statics/js/components/inject-form.js [hunk 1]
```diff
@@ -194,6 +212,12 @@ Vue.component('inject-form', {
       }
     },
 
+    getMAC: function(node) {
+      md = node.metadata;
```
Comment (safchain) [inline statics/js/components/inject-form.js]: var is missing, that said return node.metadata.MAC || "" should work

File: statics/js/components/inject-form.js [hunk 2]
```diff
@@ -127,15 +137,21 @@ Vue.component('inject-form', {
...
     error: function() {
       if (!this.node1 || !this.node2) {
-          return "Source and destination interfaces must be selected";
-      } else if (this.srcIP == "" && this.dstIP == "") {
-          return "Source and Destination IPs need to be given by user";
-      } else if (this.srcIP == "") {
-          return "Source IP need to be given by user";
-      } else if (this.dstIP == "") {
-          return "Destination IP need to be given by user";
+        return "Source and destination interfaces must be selected";
```
Comment (safchain) [inline statics/js/components/inject-form.js]: I think all the "else if" are useless as we "return"

Comment (masco-enovance) [reply statics/js/components/inject-form.js]: it is used when we select an interface which doesn't have IP/MAC and the values are not given by a user too.

**Comment by safchain** [comment]
rerun unit-tests

**Pull request merged**